oak-lucene/src/main/java/org/apache/jackrabbit/oak/plugins/index/lucene/LuceneIndexMBean.java

```diff
@@ -26,19 +25,18 @@
 import org.apache.jackrabbit.oak.api.CommitFailedException;
 import org.apache.jackrabbit.oak.api.jmx.Description;
 import org.apache.jackrabbit.oak.api.jmx.Name;
+import org.apache.jackrabbit.oak.plugins.index.search.IndexMBean;
 
-public interface LuceneIndexMBean {
+public interface LuceneIndexMBean extends IndexMBean {
     String TYPE = "LuceneIndex";
 
-    TabularData getIndexStats() throws IOException;
-
```

Comment:
Just confirming - since these are moved to the base interface - IndexMBean - this wouldn't require a major version bump in the package versioning - right ? Since the contract effectively stays the same.

Reply:
I have reverted the mbean changes. They will be moved to a separate PR.